src/main/kotlin/app/revanced/patches/spotify/disable_capture_restriction/bytecode/patch/DisableCaptureRestrictionBytecodePatch.kt

```diff
@@ -0,0 +1,53 @@
+package app.revanced.patches.spotify.disable_capture_restriction.bytecode.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.impl.BytecodeData
+import app.revanced.patcher.extensions.instruction
+import app.revanced.patcher.extensions.replaceInstruction
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.patch.impl.BytecodePatch
+import app.revanced.patcher.util.proxy.mutableTypes.MutableMethod
+import app.revanced.patches.spotify.disable_capture_restriction.annotation.DisableCaptureRestrictionCompatibility
+import app.revanced.patches.spotify.disable_capture_restriction.fingerprints.DisableCaptureRestrictionAudioDriverFingerprint
+import app.revanced.patches.spotify.disable_capture_restriction.resource.patch.DisableCaptureRestrictionResourcePatch
+import org.jf.dexlib2.iface.instruction.Instruction
+import org.jf.dexlib2.iface.instruction.OneRegisterInstruction
+
+@Patch
+@Name("disable-capture-restriction")
+@DependsOn([DisableCaptureRestrictionResourcePatch::class])
+@Description("Allow capturing Spotify's audio output while screen sharing or screen recording")
+@DisableCaptureRestrictionCompatibility
+@Version("0.0.1")
+class DisableCaptureRestrictionBytecodePatch : BytecodePatch(
+    listOf(
+        DisableCaptureRestrictionAudioDriverFingerprint
+    )
+) {
+    private fun MutableMethod.replaceConstant4Instruction(index: Int, instruction: Instruction, with: Int) {
+        val register = (instruction as OneRegisterInstruction).registerA
+        this.replaceInstruction(
+            index, "const/4 v$register, 0x${"%x".format(with)}"
```

Comment:
It is not required to format smali constants as hexadecimal, you can instead simply use a decimal here